test/probes/http-common.js

```diff
@@ -199,7 +187,7 @@ describe(`probes.${p}`, function () {
         axios(
           `${p}://localhost:${port}/foo?bar=baz`,
           function (error, response, body) {
-            expect(response.headers).exist
+            expect(response.headers).exist()()
```

Comment:
2 pairs of parenthesis?

Reply:
sorry for my ignorance

Reply:
Yes, possible, but in this case, a typo. Good catch. Thanks. Fixed.